website-warmup/task.json

```
@@ -70,18 +71,42 @@
             "helpMarkDown": "Ignore when the website has an error and continue"
         },
         {
-            "name": "BasicAuthUser",
+            "name": "Authentication",
+            "type": "pickList",
+            "label": "Select authentication type",
+            "required": false,
+            "defaultValue": "none",
+            "groupName": "auth",
+            "options": {
+                "basic": "basic",
```

Can you update this to Basic (capital B)
```suggestion
                "basic": "Basic",
```